code/modules/paperwork/paper.dm

```diff
@@ -54,7 +54,7 @@
 
 /obj/item/paper/update_icon()
 	..()
-	if(icon_state == "paper_talisman")
+	if(istype(src, /obj/item/paper/talisman))//let's not override talisman sprites, tyvm
```

Comment:
This should be overriden in the talisman’s update_icon instead.

Reply:
You're right, I should just improve the code when I see such stuff instead of going with it, lmao. Will fix tomorrow.

Reply:
Should be fixed now